.github/workflows/linux.yml

```diff
@@ -65,6 +73,18 @@ jobs:
         run: python3 ./mach test-unit --release --with-${{ env.LAYOUT }}
       - name: Lockfile check
         run: ./etc/ci/lockfile_changed.sh
+      - name: Package
+        run: python3 ./mach package --release
+      - name: Upload Package
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ env.PACKAGE }}
+          path: target/release/servo-tech-demo.tar.gz
+      - name: Upload
+        if: ${{ inputs.upload }}
+        run: python3 ./mach upload-nightly ${{ env.PACKAGE }} --secret-from-environment
+        env:
+          S3_UPLOAD_CREDENTIALS: ${{ secrets.S3_UPLOAD_CREDENTIALS }}
       - name: Package binary
```

Comment:
Do we still need this step and the next one (Archive binary)?

Reply:
I am not sure, but quick inspection of [python/servo/package_commands.py](https://github.com/servo/servo/blob/master/python/servo/package_commands.py) shows that command does not just zip files, thus I imagined that they are different. I do not know if we could/should run WPT tests on packaged builds vs. raw builds so I did not change it. @jdm may know what was rationale behind current approach.

Reply:
Thanks for the explanation! I guess we can keep both artifacts for now and decide later if switching tests to the packaged artifacts makes sense.